[DOCS] Updating approle docs and token partial to include batch token prefer… (#26490)
hashicorp/vault: website/content/docs/auth/approle.mdx

Vault-defined _roles_. The open design of `AppRole` enables a varied set of
workflows and configurations to handle large numbers of apps. This auth method
is oriented to automated workflows (machines and services), and is less useful

> for human operators. You should use `batch` tokens with this auth method as a recommended best practice.

Reviewer: ```suggestion
for human operators. We recommend using `batch` tokens with the
 `AppRole` auth method.
```
Style correction: avoid "this" as a pronoun, especially across sentences
